lib/datadog/appsec/contrib/active_record/instrumentation.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module ActiveRecord
+        # AppSec module that will be prepended to ActiveRecord adapter
+        module Instrumentation
+          module_function
+
+          def detect_sqli(sql, adapter_name)
```

Comment:
would you be open to name it `detect_sql_injection` for people outside of ASM because I spent a lot of time trying to guess what's sqli is? 😅

Reply:
yes, that's a great point, thanks! I got so used to it, that I don't notice anymore that it's actually an abbreviation

Reply:
[daa831f](https://github.com/DataDog/dd-trace-rb/pull/4167/commits/daa831f08b370a4ea965a6f66eb45d1b5a7caebc)